code/modules/mob/living/carbon/human/life.dm

```diff
@@ -119,13 +119,12 @@
 							   "qM blew my points on [pick("cOMbat Shtogun","inSuLated gloves","LOTS MASSHEEN!")]",
 							   "EI'NATH!",
 							   "WAKE UP SHEEPLES!",
-							   "et wus my [pick("wittle brother!!","fiancee","friend staying over","entiRe orphanage","love interest","wife","husband","liTTle kids","sentient cAT","accidentally")]!")
-
-			var/list/s2 = list("FUS RO DAH",
+							   "et wus my [pick("wittle brother!!","fiancee","friend staying over","entiRe orphanage","love interest","wife","husband","liTTle kids","sentient cAT","accidentally")]!",
+							   "FUS RO DAH",
 							   "fuckin tangerines!!!",
-							   "stat me",
-							   ">my face",
-							   "roll it easy!",
+							   "stAT ME",
+							   "my FACE",
```

Comment:
This one's supposed to be a 4chan bit, like >mfw, so the > might be worth hanging onto, though I think it might actually get filtered out?

Reply:
Yeah the > gets filtered out and just says "My face"